indico/modules/events/registration/client/styles/regform.module.scss

```diff
@@ -136,6 +138,78 @@
   }
 }
 
+.regform-table {
```

Comment:
Maybe rename this to something like `choices-table`

Reply:
Also stuff that's only used in choices could go into a new `ChoicesSetup.module.scss` css module